pkg/github/issues.go

```diff
@@ -67,6 +75,37 @@ func GetIssue(getClient GetClientFn, t translations.TranslationHelperFunc) (tool
 				return mcp.NewToolResultError(fmt.Sprintf("failed to get issue: %s", string(body))), nil
 			}
 
+			// Get sub-issues
+			url := fmt.Sprintf("repos/%v/%v/issues/%d/sub-issues", owner, repo, issueNumber)
```

Comment:
Please add a comment here linking to https://github.com/google/go-github/issues/3546 to indicate to future maintainers why we are doing this manually.

Reply:
Fix